api/envoy/service/status/v3/csds.proto

```diff
@@ -53,6 +53,24 @@ enum ConfigStatus {
   ERROR = 4;
 }
 
+// Config status from a client-side view.
+enum ClientSideConfigStatus {
```

Comment:
Suggest calling this `ClientConfigStatus`.

Reply:
Changed to `ClientConfigStatus`.

In this proto definition, there is the `ClientConfig` message. It might be a bit confusing to users that why `ClientConfigStatus` is not the status for the `ClientConfig` message, but the client-side view of config status. On the other hand, the `ClientSideConfigStatus` is not much better.

Better naming is still welcomed. Picking `ClientConfigStatus` since it is simpler.